scripts/ci-circleci-start-pipeline.py

```diff
@@ -66,13 +67,19 @@ def Main():
     if not created:
         created = datetime.datetime.utcnow().isoformat()
 
-    params = {
-        "mapbox_android_upstream": True,
-        "mapbox_slug": args.origin_slug,
-        "mapbox_android_hash": args.hash,
-        "mapbox_merge_main": args.current_branch == MAIN_BRANCH_NAME,
-        "benchmark_date": created
-    }
+    if args.sync_internal:
+        params = {
+            "mapbox_slug": args.origin_slug,
+            "mapbox_sync_branch": args.current_branch
```

Comment:
Only send relevant parameters to avoid declaring them in the upstream Circle config